tests/fixtures/eea_v2.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from pyaerocom.io import ReadEEAAQEREP_V2
+
+
+@pytest.fixture(scope="session")
+def eea_v2_subset_reader() -> ReadEEAAQEREP_V2:
+    return ReadEEAAQEREP_V2("EEA_AQeRep.v2.Subset")
```

Comment:
Will this fixture use or multiple modules?
If it will only be used in on `tests/io/test_read_eea_aqerep_v2.py` it might be better to define it on that module.

Reply:
I realise now that I was not fit yesterday. Sorry for that.
Fixture removed.